docs/cache/learn/how-it-works/read-concern.md

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 4
+sidebar_label: Read Concern
+title: How to use Read Concern
+description: Learn about the `ReadConcern` option for Cache Clients, and how it can be used to guarantee consistency and save costs
+---
+
+# What is Read Concern?
+
+By default, a Momento Cache follows an eventual consistency model. That is, if you write a value from the cache, and then quickly read it back within a small fraction of a second, there is a slim, but possible chance that the response will not reflect the most recently updated value. Momento clients provide a configuration option `ReadConcern`, which can be used to configure this behavior. If a strongly consistent cache is necessary, or your system is flexible in regard to consistency, the read concern header can be modified accordingly:
+
+
+| Read Concern                  | Cost Multiplier | Default |
```

Comment:
This makes it sound like it's dollar-value cost. But AFAIK it's actually throttling-operation-cost. So I think we will need to add some verbiage above this to clarify that.

cc: @kvcache @tvald

Reply:
`Operation Count Multiplier` sounds more specific than `Cost Multiplier` to me.